<a href="https://colab.research.google.com/github/mgmills93/AEM_Spring_2019/blob/master/SecureCommunities_FinalProject/Data_Analysis/FinalProject_Border_ANALYSIS.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

###Question 3: Do border states deport more individuals through SC than interior states?
---
To answer this question, I will first look at the mean arrests of border states and interior states to get an idea of how different the numbers appear. 

Then, I will group border states and interior states into border_yes and border_no respectively, and run a t-test to determine whether the mean number of arrests for both groups is significantly different.

In [22]:
import pandas as pd
import numpy as np
import scipy
from scipy import stats

border_sums = pd.read_csv('https://raw.githubusercontent.com/mgmills93/AEM_Spring_2019/master/SecureCommunities_FinalProject/Datasets/border_states.csv')
clean_border = border_sums.drop(columns=['Unnamed: 0'])
clean_border.groupby('Border').mean()




Arrest_Number
Border               
N              1178.1
Y              6831.8

In [23]:
border_yes = clean_border.loc[clean_border['Border'] == 'Y', 'Arrest_Number']
border_no = clean_border.loc[clean_border['Border'] == 'N', 'Arrest_Number']
#print(border_yes)
#print(border_no)

twosample_results = scipy.stats.ttest_ind(border_yes, border_no)

print(twosample_results)

Ttest_indResult(statistic=2.44228771872102, pvalue=0.017962945918700873)


###Analysis
---
The mean number of arrests for border states was 5653.7 greater than for interior states, which is what I would expect to see (ICE presence is likely much greater in states that share a border with a foreign country versus those that don't).

The t-statistic from my t-test was 2.44, which suggests a significant difference in the mean arrests beteween border states and interior states.